pkg/addon/addon.go

```diff
@@ -498,6 +498,9 @@ func RenderApplication(addon *types.Addon, args map[string]interface{}) (*v1beta
 	}
 	for _, tmpl := range addon.CUETemplates {
 		comp, err := renderCUETemplate(tmpl, addon.Parameters, args)
+		if addon.Name == "observability" && strings.HasSuffix(comp.Name, ".cue") {
+			comp.Name = strings.Split(comp.Name, ".")[0]
+		}
```

Comment:
Please put this after error handling, or may cause nil pointer panic.

Reply:
Correct. Thanks for the reminder. Fixed it.